.github/workflows/quickstart.yaml

```diff
@@ -0,0 +1,77 @@
+name: Build and Deploy
+
+on: [push, pull_request]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21
+
+      - name: Install dependencies
+        run: |
+          # Install kind
```

Comment:
Would it make more sense to use https://github.com/helm/kind-action instead of manually doing this ourselves?

Reply:
I'm not sure if it would be possible to load a container image with the GitHub Action
```
kind load docker-image checkpoint-restore-operator:ci
```